src/auth/index.js

```diff
@@ -0,0 +1,50 @@
+import { Router } from 'express';
+
+import { requireFields, ErrorHandler, ValidationException } from 'utils/validation';
+import passport, { authenticate } from './passport';
+
+const router = Router();
+
+router.post('/login', requireFields('email', 'password'),
+  (req, res, next) => {
+    const handleError = ErrorHandler(res, next);
+
+    authenticate('local-login', req, res, next)
+      .then(({ id, email, role }) => res.status(200).send({ id, email, role }))
+      .catch(handleError);
+  });
+
+router.post('/register', requireFields('email', 'password'),
+  (req, res, next) => {
+    const handleError = ErrorHandler(res, next);
+    const { password, confirmPassword } = req.body;
+
+    if (password.length < 6) {
+      return handleError(new ValidationException({ password: 'Password must contain at least 6 characters' }));
+    }
+
+    if (password !== confirmPassword) {
```

Comment:
no need to send "password confirmation" to the server. We only need it on client for user not to mistype.